blockly-override.js

```diff
@@ -238,46 +256,99 @@ Blockly.setPhantomBlock = function (connection, targetBlock) {
         position.y = xy.y - connection.y_ - (targetConnection.y_ - targetBlock.getBoundingRectangle().topLeft.y);
     }
 
-    phantomSvgPath.setAttribute('d', targetBlock.svgPath_.getAttribute('d'));
-    phantomSvgPath.setAttribute('fill', targetBlock.getColour());
-    phantomSvgPath.setAttribute('fill-opacity', 0.25);
-    phantomSvgPath.setAttribute('stroke', targetBlock.getColour());
-    phantomSvgPath.setAttribute('stroke-width', 1);
-    phantomSvgPath.setAttribute('stroke-dasharray', 6);
+    const phantomSvgGroup = Blockly.utils.getPhantomSvgGroup(targetBlock, position);
 
-    phantomSvgGroup.appendChild(phantomSvgPath);
-    phantomSvgGroup.appendChild(phantomSvgText);
-    phantomSvgGroup.setAttribute('transform', `translate(${position.x}, ${position.y})`);
+    sourceBlock.svgGroup_.appendChild(phantomSvgGroup);
+    if (animationSupported) {
+        phantomSvgGroup.animate([{
+            opacity: 0,
+        }, {
+            opacity: 1,
+        }], {
+                duration: 400,
+                easing: 'ease-out',
+            });
+        breathingAnimation = phantomSvgGroup.animate([{
+            opacity: 0.7,
+        }, {
+            opacity: 1,
+        }, {
+            opacity: 0.7,
+        }], {
+                delay: 400,
+                duration: 1200,
+                easing: 'ease-in-out',
+                iterations: Infinity,
+            });
+    } else {
+        phantomSvgGroup.style.opacity = 1;
+        breathingAnimation = function () {
+            phantomSvgGroup.style.opacity = 1;
+        };
+        breathingAnimation.cancel = function () {
+            return null;
+        };
+    }
 
-    Blockly.removePhantomBlock();
 
-    sourceBlock.svgGroup_.appendChild(phantomSvgGroup);
+    Blockly.phantomBlock_ = {
+        svgRoot: phantomSvgGroup,
+        position,
+        animation: breathingAnimation,
+    };
+};
+
+Blockly.setPhantomBlockByPosition = function (workspace, targetBlock) {
+
+    let breathingAnimation;
+
+    let position = {
+        x: 200,
+        y: 200,
+    }
+
+    // Get the C-Blocks from the workspace.
```

Comment:
This logic should probably be inside kano-code. This would make it easier to change it later on. And kwc-blockly should not have opinions on how where those blocks show, but just show it where it's asked to